brainframe/cli/dependencies.py

```diff
@@ -66,7 +66,18 @@ def _install_docker():
 docker_compose = Dependency(
     "docker-compose",
     "install.ask-install-docker-compose",
-    lambda: os_utils.run(["pip3", "install", "docker-compose"]),
+    lambda: os_utils.run(
+        [
+            "pip3",
+            "install",
+            # Avoids warning message about touching the non-root user's cache
+            "--no-cache-dir",
+            # Installs globally even if the user has docker-compose installed
+            # locally
+            "--ignore-installed",
```

Comment:
Doesn't pip default to using `--user` implicitly now?

Reply:
Pip has kind of strange behavior when run in a sudo environment. It will inspect the normal user's local installation environment in `~/.local` and not install packages globally if they're installed locally there. This flag is to prevent that behavior.